addons/point_of_sale/static/src/xml/Screens/ClientListScreen/ClientListScreen.xml

```diff
@@ -52,6 +52,7 @@
                             <div class="subwindow-container collapsed">
                                 <div t-if="state.detailIsShown" class="client-details-contents subwindow-container-fix">
                                     <ClientDetailsEdit t-props="state.editModeProps"
+                                                       changes="changes"
```

Comment:
Can you try putting `changes` inside `editModeProps`? Because if it works, we won't need this change.